bve-corpus/src/options.rs

```diff
@@ -35,16 +35,18 @@ impl FromStr for FileType {
 }
 
 #[derive(Clone)]
+#[allow(clippy::struct_excessive_bools)]
 pub struct Options {
     pub help: bool,
-    /// Location of root of bve folder
```

Comment:
Wait why 

They weren't hurting anyone, let the comments live their lives in peace

Reply:
Because they aren't in sync with the help message, and for now I would rather have them be in the help message than in two places.